price-feeder.example.toml

```diff
@@ -14,6 +14,10 @@ threshold = "1.5"
 base = "ATOM"
 threshold = "1.5"
 
+[[deviation_thresholds]]
```

Comment:
Since this is getting backported should we do these sample config updates separately?

Reply:
Oh yeah thats right, took out USK👍